jac-uk/apply
# #936 Refresh of Vacancy Page

## What's included?
Update the UI of vacancies and vacancy details page when a candidate is not signed into the platform.

https://www.figma.com/proto/toOjVICtIYu5WmNbhJuyxm/JAC-Candidate-Profile?page-id=146%3A1970&node-id=1093%3A4988&viewport=-2547%2C1394%2C0.08&scaling=min-zoom&starting-point-node-id=1093%3A4988

Note: the `BackToTop` button will be applied to all pages to have a smooth scroll transition in the apply site. 

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
Develop site: https://jac-apply-develop--pr939-feature-936-refresh-o1y9n24p.web.app
Staging site: https://jac-apply-staging--pr939-feature-936-refresh-yzsk3ino.web.app

Vacancies:
1. Go to vacancies page.
2. Check if the information shows properly in different `advert types`.
3. Check if the search functionality works properly.

Vacancy details:
1. Select a vacancy and click `Vacancy details`.
2. Check if the information shows properly in the vacancy details.
3. Check if the `Print all information`, `View more`, and `View less` work properly.

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
Screenshot of vacancies:

![image](https://user-images.githubusercontent.com/79906532/206175442-776b7a78-6aa7-4448-b602-908a457498e6.png)

Screenshot of vacancy details:

![Vacancy-details-Judicial-Appointments-Commission](https://user-images.githubusercontent.com/79906532/212894147-81e7129a-9c2e-4eec-b93b-fe067d8e53c7.png)

---
PREVIEW:DEVELOP
